docs/ARCHITECTURE.md

```diff
@@ -16,15 +16,12 @@ A rough structure of the build is as follows
 
 #### Regenerate Jobs
 
-Jenkins CI jobs are generated by the [regeneration pipelines](https://ci.adoptopenjdk.net/job/build-scripts/job/utils/) in Jenkins. The documentation for this process can be found at [regeneration docs](https://github.com/adoptium/temurin-build/tree/master/pipelines/build/regeneration), however they generate the jobs
-at <https://ci.adoptopenjdk.net/job/build-scripts/> which are the builds that coordinate the entire process of nightly
-and release builds.
+Jenkins CI jobs are generated by the Jenkins [regeneration pipelines](https://ci.adoptopenjdk.net/job/build-scripts/job/utils/). Generated at <https://ci.adoptopenjdk.net/job/build-scripts/> and their builds coordinate the entire process of nightly and release builds.
```

Comment:
I'm still not clear on what the second sentence is telling me.

Reply:
@karianna 
would rather say "and their builds populate the nightly and release builds "   ?

Reply:
That's better yes, I also just realised what the "Generated at <https://ci.adoptopenjdk.net/job/build-scripts/>" actually means, that's the location of where the generated jobs are created.  I'll make a proposed commit.